tests/src/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -168,10 +230,9 @@ public int RunTest(string executable, string outputFile, string errorFile)
 
             // Check if we are running in Windows
             string operatingSystem = System.Environment.GetEnvironmentVariable("OS");
-            bool runningInWindows = (operatingSystem != null && operatingSystem.StartsWith("Windows"));
 
             // We can't yet take crash dumps on non-Windows OSs for timed-out tests
```

Comment:
This comment is no longer true with this change, right?  This change will give us crash dumps on timed out tests on Linux?